README.md

```diff
@@ -1,15 +1,9 @@
-# pliers
```

Comment:
If you want to be even more radical (and recycle the README.md in the docs, you can convert the [readme to rest](https://pandoc.org/try/?text=&from=markdown&to=rst&standalone=0) and include the [README.rst in index.rst like fmriprep does](https://github.com/nipreps/fmriprep/blame/4bb8a61cde27575865cdd2b7df5afcb5d6860523/docs/index.rst#L7)

Reply:
That's basically what I was too lazy to do but its a good idea